clock.json

```diff
@@ -249,6 +293,54 @@
             "expected": true
          },
          {
+            "description": "clocks with hour overflow by several days",
+            "clock1": {
+               "hour": 3,
+               "minute": 11
+            },
+            "clock2": {
+               "hour": 99,
+               "minute": 11
+            },
+            "expected": true
+         },
+         {
+            "description": "clocks with negative hour",
+            "clock1": {
+               "hour": 22,
+               "minute": 40
+            },
+            "clock2": {
+               "hour": -2,
+               "minute": 40
+            },
+            "expected": true
+         },
+         {
+            "description": "clocks with negative hour that wraps",
```

Comment:
could you explain the difference between this and the previous case "clocks with negative hour"? It doesn't look like they are different enough to warrant a separate case since both negative values are < -24, unless this is testing something different